### ISSUE TITLE
Feature request: Make navigation fixed

### ISSUE THREAD

**Issue by enventa** [tareq1988/wedocs-plugin#77]
Hi @tareq1988 !

Great plugin, thanks! I would like to ask you to consider making the sidebar navigation fixed when scrolling down. It would make it easier to navigate from article to article or from section to article.

Thanks!

**Comment by lparkyn** [comment]
I agree. I would like to see fixed sidebar navigation as well. This would greatly speed up navigation within a document and improve the reader experience.
